new-lamassu-admin/src/pages/Wizard/components/Wallet/ChooseCoin.js

```diff
@@ -38,7 +38,7 @@ const ChooseCoin = ({ addData }) => {
     if (!schema.isValidSync(it)) return setError(true)
 
     if (it.coin !== 'BTC') {
-      return addData({ coin: it.coin, zeroConf: 'all-zero-conf' })
+      return addData({ coin: it.coin, zeroConf: 'none', zeroConfLimit: 0 })
     }
```

Comment:
We might be able to remove this since we set default the same values for all coins, am I right? @RafaelTaranto

Reply:
Isn't this where we're setting the default?